react-function-components/src/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const CustomButton = () => {
+  return <button>Click Me!</button>;
+};
+
+const element = <CustomButton />;
+
+ReactDOM.render(
+  element,
```

Comment:
No real need to save it into a variable first, you can just directly pass the JSX element to `render`